frontend/src/js/components/IngestionEvents/IngestionEvents.tsx

```diff
@@ -63,14 +63,23 @@ const getBlobStatus = (statuses: ExtractorStatus[]) => {
     return failures.length > 0 ? blobStatusIcons.completeWithErrors : inProgress.length > 0 ? blobStatusIcons.inProgress : blobStatusIcons.complete
 }
```

Comment:
I know this is not related to this PR but just pointing it out to see if we need to fix this. There are sometimes failures on a blob extractor because of previous uploads. For example, I upload a file and it failes to complete OcrMyPdf, but the I upload the file again, and it succeeds. The statuses are showing the list of all statuses for all the uploads for this file. And while the file was processed successfully in the second time, dashboard is  showing this record as a failed record. Is that intentional?
![image](https://github.com/guardian/giant/assets/15894063/85558e97-a940-493e-bce1-dae71f3a2dd4)

Reply:
We discussed this in a mobbing session. This is a user experience concern and we will decide later if it needs to behave differently